core/domain/stats_domain_test.py

```diff
@@ -812,14 +812,14 @@ def test_cannot_update_exp_issue_with_no_schema_version(self):
         exp_issues_model = stats_models.ExplorationIssuesModel.get_model(
             'exp_id', 1)
 
-        with self.assertRaises(Exception):
+        with self.assertRaisesRegexp(Exception, ''):
```

Comment:
Ditto

Reply:
I don't see this fixed!

Reply:
Look again

Reply:
@Hudda, Sorry I was looking through the filename link of the comment thread and ended up looking through an old commit changes, I hope you understand. I'll make sure to check the commit id in the link before commenting, I think that will help me to avoid this problem and I won't repeat it! :)   (I've also mentioned this [here](https://github.com/oppia/oppia/pull/9860#pullrequestreview-450305239))